src/installer/pkg/projects/nativeaot-packages.proj

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <Import Sdk="Microsoft.Build.Traversal" Project="Sdk.props" />
+
+  <ItemGroup>
+    <Project Include="Microsoft.DotNet.ILCompiler\Microsoft.DotNet.ILCompiler.pkgproj" />
+    <ProjectReference Include="@(Project)" />
+    <ProjectReference Include="@(Project)" AdditionalProperties="PackageTargetRuntime=$(PackageRID)" />
```

Comment:
Why do we need to include it once without the additional properties and then with it?

Reply:
One builds the "lineup package" (or what's the right term - the one without the `runtime.$RID` prefix), the other builds the RID-specific package (with the `runtime.$RID` prefix).

Reply:
Ah cool.